src/legacy/status_im/data_store/settings.cljs

```diff
@@ -51,3 +51,10 @@
       (visibility-status-updates/<-rpc-settings)
       (set/rename-keys {:compressedKey :compressed-key
                         :emojiHash     :emoji-hash})))
+
+(defn rpc->setting-value
```

Comment:
is there any reason we need to have this in legacy folder? seems like something we can move to profile/datastore.cljs ?

Reply:
and start the migration! :)

Reply:
> is there any reason we need to have this in legacy folder? seems like something we can move to profile/datastore.cljs ?

I placed it in the `legacy.*` ns as the RPC data helpers for settings are used from it.
https://github.com/status-im/status-mobile/blob/9ff70ed9900336ce782169979c8d9922a0709bb8/src/status_im/contexts/profile/login/events.cljs#L86
https://github.com/status-im/status-mobile/blob/9ff70ed9900336ce782169979c8d9922a0709bb8/src/legacy/status_im/data_store/settings.cljs#L38

Moving the data-store for settings to `status-im.*` ns would be easy when we do it.

cc @flexsurfer